modules/ossm-migrating-reference-smcp-configurations.adoc

```diff
@@ -0,0 +1,448 @@
+// Module included in the following assemblies:
+//
+// * service-mesh-docs-main/migrating/reference/ossm-migrating-reference.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="ossm-migrating-reference-smcp-configurations_{context}"]
+= Configuration fields mapping between Service Mesh 2 and Service Mesh 3
+
+Many of the `ServiceMeshControlPlane` `spec.` configuration fields in {SMProductName} 2 can be configured in the `Istio` resource in {SMProduct} 3.
```

Comment:
Having a tough time parsing this:

>  `ServiceMeshControlPlane` `spec.` configuration fields 

Consider alternative phrasings to avoid multiple backticked items in sequence.